### PR TITLE
Stiffness and damping coefficients

### LINKED ISSUE #435: Add damping coefficients to fluid flow
This issue came from issue #261 , being divided into a smaller problem. In order to close it, we need to add the numerical damping coefficients and to implement a test showing that the results match the expected.

To calculate the damping coefficients, the authors suggest a small perturbation in the journal center velocities to do so. However, we are not sure how to implement this perturbation. Because our rotor works in steady state, these velocities are zero (it's not moving). Some attempts were made on how to calculate the damping matrix and compared it with Friswell's non-numerical way:

- Attempt 1- Changing angular frequency;
- Attempt 2- Changing angular frequency, but replacing the rotor velocity with the fluid velocity;
- Attempt 3- Changing angular frequency, keeping the rotor in same position, replacing the rotor velocity with the fluid velocity.

The results of these attempts are recorded in the old issue #261  and none have been successful.

We continue to study the references on the subject. I think that to find the damping matrices it is necessary to consider the time variable in the equations. A more practical idea is to initially try to implement a model already ready and well accepted in the literature, without deducting the equations from the beginning.

### PR DESCRIPTION
**Pull request by JuliaMota**
This pull request closes issue #435 . It is now possible to numerically calculate the stiffness and damping coefficients in the bearings.

Close #435

### CODE REVIEW

File: ross/available_materials.toml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 [Materials.Steel]
 name = "Steel"
-rho = 7810
+rho = 7810.0
 E = 211000000000.0
-Poisson = 0.2992610837438423
 G_s = 81200000000.0
+Poisson = 0.2992610837438423
```
Comment (raphaeltimbo) [inline ross/available_materials.toml:6]: I think these changes are not related to this PR.
Maybe this file got changed when running the doctests.
When running doctests we are creating several files and also changing the available_materials file. 
I have open an issue for that #590.

**Comment by rodrigomoliveira1** [comment]
@JuliaMota can we merge this PR?
These failing tests are not related to the correct functioning of the code. Travis CI - Branch is failing because it's getting an old version, without the correct pytest dependacy. AppVeyor, is bugging for some reason I don't know yet (build not found or access denied).

**Comment by JuliaMota** [comment]
@rodrigomoliveira1 , I believe we can. I did all the tests on my computer and there was no problem. I didn't quite understand what happened.
